<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>New_person_dialog</class>
 <widget class="QDialog" name="New_person_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="edit_name_label">
           <property name="text">
            <string>Jméno: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="edit_name"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="edit_surname_label">
           <property name="text">
            <string>Přijímení: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="edit_surname"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="edit_day_label">
           <property name="text">
            <string>Den narození: </string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="edit_day"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="edit_month_label">
           <property name="text">
            <string>Měsíc:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="edit_month"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="edit_year_label">
           <property name="text">
            <string>Rok:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="edit_year"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCalendarWidget" name="calendarWidget"/>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="button_close">
         <property name="text">
          <string>Zavřít</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_OK">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
